sourcetools/j9constantpool/com/ibm/oti/VMCPTool/FieldRef.java

```diff
@@ -124,6 +124,9 @@ protected String fieldType() {
 		switch (((Alias) primary).nas.signature.data.charAt(0)) {
 		case '[':
 		case 'L':
+		/*[IF INLINE-TYPES]*/
+		case 'Q':
+		/*[ENDIF] INLINE-TYPES */
```

Comment:
This file is not run through the preprocessor, so these are just comments and can be removed. We'll need to ensure that any `Q` signatures in `vmconstantpool.xml` are targeted at a version that supports inline types.

Reply:
Ok, it looks like there are no `Q` signatures in `vmconstantpool.xml` right now, which I assume means I don't need to do any version targeting?

Reply:
I think we should add support for `Q` signatures here now. If and when we need to use them in `vmconstantpool.xml`, those uses can be guarded by the version or a flag.

Reply:
Lines 127 and 129 should simply be deleted.

Reply:
whoops, I thought there was only one of them. fixed in [89dde4a](https://github.com/eclipse-openj9/openj9/commit/89dde4a1efae9e5a1c1a1991631067030591c6ee)